src/IssueViz.Security/HotspotsList/ViewModels/HotspotViewModel.cs

```diff
@@ -78,6 +80,12 @@ public int Column
         public string DisplayPath =>
             Path.GetFileName(Hotspot.CurrentFilePath ?? ((IHotspot)Hotspot.Issue).ServerFilePath);
 
+        public string CategoryDisplayName =>
+            SecurityCategoryDisplayNames.Mapping.TryGetValue(
+                ((IHotspot) Hotspot.Issue).Rule.SecurityCategory, out var displayName)
+                ? displayName
+                : string.Empty;
```

Comment:
I'm not sure _Empty_ is the most useful value we could return here. 
The short forms are fairly readable on their own, so returning that in the hopefully very rare case of us encountering a new mapping would at least give the user some useful information.
Alternatively, something like `{unrecognized}` or `{unknown}` would provide the user with some information about why the category name is not displayed.

Reply:
I was wary of using the unfriendly name, as that might cause users to ignore it and we will miss the potential problems.
{unknown} sounds good 👍